WooCommerce/Classes/ViewRelated/Filters/FilterListViewController.swift

```diff
@@ -281,6 +284,33 @@ private extension FilterListViewController {
                                                                                                      viewModel: productSelectorViewModel))
                 }()
                 self.listSelector.navigationController?.present(controller, animated: true)
+
+            case .customer(let siteID):
+                let addressFormViewModel = CreateOrderAddressFormViewModel(
+                    siteID: siteID,
+                    addressData: .init(billingAddress: nil, shippingAddress: nil),
+                    onAddressUpdate: nil
+                )
+
+                let controller: CustomerSelectorViewController = {
+                    return CustomerSelectorViewController(
+                        siteID: siteID,
+                        configuration: .configurationForOrderFilter,
+                        addressFormViewModel: addressFormViewModel,
+                        onCustomerSelected: { [weak self] customer in
+                            selected.selectedValue = CustomerFilter(customer: customer)
+
+                            self?.updateUI(numberOfActiveFilters: self?.viewModel.filterTypeViewModels.numberOfActiveFilters ?? 0)
+                            self?.listSelector.reloadData()
+                            self?.listSelector.navigationController?.dismiss(animated: true)
+                        }
+                    )
+                }()
+
+                self.listSelector.navigationController?.present(
+                    WooNavigationController(rootViewController: controller),
+                    animated: true
+                )
```

Comment:
I might have missed this when reviewing the product filter but we should present the customer list from `listSelector` for simplicity:

```suggestion
                self.listSelector.present(
                    WooNavigationController(rootViewController: controller),
                    animated: true
                )
```

The dismissal should be updated accordingly too.

Reply:
Updated dismissals in https://github.com/woocommerce/woocommerce-ios/pull/12100/commits/2ddd777ce3b74abba02bbb53a0deb9fd43163b44